docs/dev/api/storage.md

```diff
@@ -405,6 +405,220 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 ```
 </details>
 
+---
+### Get App Storage Group Settings
+
+<details><summary><span className="api get">GET</span> <code>/v1/storage/groups/{group_id}/settings</code></summary>
+<p/>
+
+Returns the settings of the requested app group (apps containing multiple files).
```

Comment:
Returns the settings of the requested app groupID

Reply:
that's even more wrong.. wrong on so many levels..

Reply:
Returns the settings of the requested app group id. would be probably best..

Reply:
agreed